packages/wonder-blocks-theming/src/utils/merge-theme.ts

```diff
@@ -0,0 +1,33 @@
+type RecursivePartial<T> = {
+    [P in keyof T]?: RecursivePartial<T[P]> | string | number | boolean;
+};
+
+/**
+ * Allow us to create a new copy of the target theme by overriding some of its
```

Comment:
teeniest tiniest nit
```suggestion
 * Allows us to create a new copy of the target theme by overriding some of its
```